test/plugins/features/movies/index.test.js

```diff
@@ -23,6 +23,36 @@ describe('movies integration', () => {
 
   });
 
+  describe('location', () => {
+
+    it('associates a location', async () => {
+      const payload = { location: 'Berkeley' };
+
+      const response = await Movies.inject({
+        url: '/movies/1/locations',
+        method: 'POST',
+        payload
+      });
+
+      expect(response.statusCode).to.eql(200);
+      expect(response.result.object).to.eql('movie');
+      expect(response.result.locations).to.include.members(['Berkeley']);
+    });
+
+    it('does not associate a location multiple times', async () => {
+      const payload = { location: 'Berkeley' };
+
+      const response = await Movies.inject({
```

Comment:
It's not immediately clear why this request fails with status code 500 when the exact same request in the test right before this succeeds with status code 200. I think it might be better to 1) add a `beforeEach()` to truncate all relevant tables and 2) explicitly make two requests in this test to show that a second request with the same movie ID causes a 500 (not the request itself).

Reply:
I like the before each approach. Cool!